docs/bazel.md

```diff
@@ -1,136 +1,100 @@
-<!-- Generated with Stardoc: http://skydoc.bazel.build -->
+# Bazel rules and macros
```

Comment:
This is the main thing to review. Best not to look at the diff, and just the file itself.